pkg/service/service.go

```diff
@@ -851,8 +851,10 @@ func (s *Service) restoreBackendsLocked() error {
 			logfields.L3n4Addr:  b.L3n4Addr.String(),
 		}).Debug("Restoring backend")
 		if err := RestoreBackendID(b.L3n4Addr, b.ID); err != nil {
-			return fmt.Errorf("Unable to restore backend ID %d for %q: %s",
-				b.ID, b.L3n4Addr, err)
+			log.WithError(err).WithField(logfields.BackendID, b.ID).
+				Warn("Unable to restore backend ID %d for %q: %s",
```

Comment:
```suggestion
				Warnf("Unable to restore backend ID %d for %q: %s",
```

Reply:
As per comment by @brb `This could help to reduce a number of connections which would be interrupted due to svc/backend ID changes which happens when the restoration fails.`